api/controllers/game/spectate.js

```diff
@@ -1,17 +1,44 @@
 module.exports = async function (req, res) {
   try {
     const { gameId } = req.body;
-    const game = await gameService.populateGame({ gameId });
+    const promiseGame = await gameService.populateGame({ gameId });
+    const promisePlayer = await userService.findUser({ userId: req.session.usr });
```

Comment:
Let's rename `promisePlayer` to `promiseSpectator` since they aren't playing in this game